lib/bandit/http2/adapter.ex

```diff
@@ -113,7 +114,6 @@ defmodule Bandit.HTTP2.Adapter do
     # (RFC7230§4.1). The whole notion of chunked encoding is moot in HTTP/2 anyway (RFC7540§8.1)
     # so this entire section of the API is a bit slanty regardless.
     send_data(adapter, chunk, chunk == <<>>)
-    :ok
```

Comment:
This is as intended. As noted elsewhere on this review, we're constrained in what we can return here by our conformance to the `Plug.Conn.Adapter` behaviour. Specifically, Plug's [explicit guidance](https://hexdocs.pm/plug/Plug.Conn.Adapter.html#c:chunk/2) here is that we return `:ok`, as we currently do. 

I agree that this is an awkward corner of the Plug API, and is one of those rare cases where things aren't entirely functional (nothing actually changes in a `Plug.Conn` struct as a result of a chunk call, even though bytes are sent out on the wire). But we're doing the right thing here already.

Reply:
Oh, sorry. I should have checked the `Plug` documentation first. 
I reverted the change to always return `:ok` here.